cmake/FileIOConfig.cmake

```diff
@@ -0,0 +1,34 @@
+#.rst:
+# FindFileIO
+# -------
+#
+# Find libFileIO
+# FileIO deals with file access and directory traversal.
```

Comment:
For this and the other LROSS pull request. I do not think the <repo name>Config.cmake are necessary. Further, they are hard to test also so I believe all of the special cmake config files should no be installed and in fact removed
(Murmurhash, FileIO, StopWatch, DeathKnell, Queuenado)